### DISCUSSION TITLE
fluxes predicted by the model are net fluxes?

### DISCUSSION THREAD

**Discussion by qaz-94** [SysBioChalmers/Human-GEM#249] [Q&A] [answered]
Hi, thank you for developing this amazing model and I am currently building the GEM for different tissues, so eventually, I would like to compare the fluxes across the tissues.  I am curious whether the output of reaction fluxes are **net fluxes**[which equal the product of the biomass abundance and the specific flux (expressed per unit biomass) ] or just simply the **unit biomass flux**?  So essentially, I am wondering, if I want to compare across the different models, **do I have to divide by biomass to account for biomass difference?**
Thank you so much! Really looking forward to your reply!

**Comment by haowang-bioinfo** [comment, marked as answer]
@qaz-94 what are the constrains (e.g. exchange fluxes, objectives) used in getting the fluxes?

**Reply by qaz-94** [reply]
I used the -1000 to 1000 for reversible and 0 to 1000 for irreversible.  Thank you

**Reply by JonathanRob** [reply]
Hi @qaz-94, I think what @Hao-Chalmers is implying is that the units of the reaction fluxes in a model will depend on the units used when constraining the model.

The fluxes in the model are in units of moles per mass dry cell weight per time, or most commonly mmol/gDW/h (gDW = grams dry cell weight). The exception is the biomass reaction (`biomass_human`), the stoichiometric coefficients of which are designed such that its flux is in units of gDW/gDW/h, or simply 1/h (corresponding to a growth rate).

I'm not sure I completely understand your question, but in general, yes, you can directly compare the fluxes among your different tissue GEMs, since they are on a per-cell-mass basis. However, since some tissues may exhibit very different biomass fluxes (assuming your objective is biomass production), it may help in some situations to normalize the fluxes of a model by the biomass flux. It depends on the question you're asking, though.
